test/testSetup.ts

```diff
@@ -0,0 +1,337 @@
+import { config } from 'dotenv';
+import request from 'supertest';
+import { AlkemioClient } from '@alkemio/client-lib';
+import { TestUser } from './utils/token.helper';
+import { getMails } from './utils/mailslurper.rest.requests';
+import { userData } from './utils/common-params';
+
+config({ path: '.env' });
+
+function delay(ms: number) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+const kratosDomain =
+  'http://localhost:3000/identity/ory/kratos/public/self-service';
+const kratosRegistrationFlowEndpoint = '/registration/api';
+const kratosVerificationFlowEndpoint = '/verification/api';
+
+const PASSWORD = process.env.AUTH_TEST_HARNESS_PASSWORD || '';
+const SERVER_URL = process.env.ALKEMIO_SERVER_URL;
+
+type KratosFlow = {
```

Comment:
As a rule of thumb, and generally accepted metric, try to keep service classes < 200 lines. This file serves multiple purposes:

- typings
- constants
- test setup
- kratos registration flow
- kratos verification flow
- utils

These could easily be split in separate files. At the moment the code is a bit hard to read, splitting it would make the overall design better.